pkgs/applications/networking/expressvpn/default.nix

```
@@ -0,0 +1,104 @@
+{ autoPatchelfHook
+, buildFHSUserEnv
+, dpkg
+, fetchurl
+, inotify-tools
+, lib
+, stdenvNoCC
+, sysctl
+, writeScript
+}:
+
+let
+  name = "expressvpn";
+  version = "3.24.0.2-1";
```

```suggestion
  version = "3.24.0.2";
```
thats the real version number. -1 is something that the deb generators append.